src/client/dfs/dfs.c

```diff
@@ -1270,16 +1324,20 @@ dfs_mount(daos_handle_t poh, daos_handle_t coh, int flags, dfs_t **_dfs)
 		D_GOTO(err_dfs, rc = daos_der2errno(rc));
 	}
 
+	entry = daos_prop_entry_get(prop, DAOS_PROP_CO_ROOTS);
+	D_ASSERT(entry != NULL);
+	roots = (struct daos_prop_co_roots *)entry->dpe_val_ptr;
+	dfs->sb_oid = roots->cr_oids[0];
+	dfs->root.oid = roots->cr_oids[1];
+	dfs->root.parent_oid = dfs->sb_oid;
```

Comment:
Do we need valid the the root id? in case it open an old version container?

Reply:
I can add a version number, but I think this is responsibility of middleware, not DAOS